### DISCUSSION TITLE
zone 15

### DISCUSSION THREAD

**Discussion by jokkling** [Dilbert66/esphome-dsckeybus#34] [Q&A]
Hello! I just found this awsome project. I'm integrating my PC1616 alarm. The communication works well so far, but I have onte question. I use other zones than the 11 that is available in the code. Is there a way to get the zone status for zone 15 and so on?

**Comment by Dilbert66** [comment]
The firmware supports 64 zones.  Just add more sections in the yaml configuration to mach your needs.

**Reply by jokkling** [reply]
Aha! I found it now. I had to add some new case-codes. Thanks!!
